aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -122,48 +122,48 @@ def process_message(record: Dict) -> Dict[str, Union[str, bool]]:
     Ensure the message is proper format. Get the customer
     entitlements and send the information to the SCC service.
     """
+    sqs_event_manager_config = Defaults.get_sqs_event_manager_config()
     result = {
         'itemIdentifier': record.get('messageId') or 'unknown',
         'status': 'unknown',
         'error': True
     }
     try:
         message = AWSSNSMessage(record)
+        auth_token = sqs_event_manager_config.get('auth_token') or ''
         if not message.action:
             result['status'] = 'No action defined in SNS message'
             logger.error(result['status'])
         elif message.action == 'entitlement-updated':
-            # Notify SCC of customer entitlement change for the given product
-            customer_id = message.customer_id
-            product_code = message.product_code
-            logger.info(
-                'requesting entitlements for customer {} and product {}'.format(
-                    customer_id, product_code
+            endpoint_url = sqs_event_manager_config['entitlement_change_url']
+            result.update(
+                send_to_scc(
+                    entitlement_updated(message), endpoint_url, auth_token
                 )
             )
-            entitlements = AWSCustomerEntitlement(
-                customer_id, product_code
+        elif message.action == 'subscribe-success':
+            endpoint_url = sqs_event_manager_config['subscribe_success_url']
```

Comment:
ditto w.r.t. direct access of the dict key.

Reply:
done